Interfaces/Cosmos.DataTransfer.Interfaces/DataItemExtensions.cs

```diff
@@ -11,10 +11,12 @@ public static class DataItemExtensions
     /// <param name="requireStringId">If true, adds a new GUID "id" field to any top level items where one is not already present.</param>
     /// <returns>A dynamic object containing the entire data structure.</returns>
     /// <remarks>The returned ExpandoObject can be used directly as an IDictionary.</remarks>
-    public static ExpandoObject? BuildDynamicObjectTree(this IDataItem? source, bool requireStringId = false)
+    public static ExpandoObject? BuildDynamicObjectTree(this IDataItem? source, bool requireStringId = false, bool ignoreNullValues = false)
```

Comment:
This isn't passed to recursive calls so currently only works for top level properties. This can be extended later.